Meta/run.sh

```diff
@@ -17,8 +17,12 @@ SCRIPT_DIR="$(dirname "${0}")"
 #SERENITY_PACKET_LOGGING_ARG="-object filter-dump,id=hue,netdev=breh,file=e1000.pcap"
 
 # FIXME: Enable for SERENITY_ARCH=aarch64 if on an aarch64 host?
-KVM_SUPPORT="0"
-[ -e /dev/kvm ] && [ -r /dev/kvm ] && [ -w /dev/kvm ] && [ "$SERENITY_ARCH" != "aarch64" ] && KVM_SUPPORT="1"
+
+# Check if KVM_SUPPORT is unset
+if [ -z ${KVM_SUPPORT+x} ]; then
```

Comment:
Since all the external/env variables that can influence this have a `SERENITY_` prefix, perhaps this should instead check for that and assign it to `KVM_SUPPORT` if it's set?